hasher-matcher-actioner/webapp/src/Api.jsx

```diff
@@ -67,6 +67,16 @@ export function fetchDashboardCardSummary(path) {
   return apiGet(`/${path}`);
 }
 
+export const StatsTimeSpans = Object.freeze({
```

Comment:
nit: comment to where this maps to a python enum? 

Also I created `utils/constants.jsx` a few PRs ago, if you think we should instead put enums/constants inline I'm open to it, I mostly just think we should try to be consistent.

Reply:
Addressed in https://github.com/facebook/ThreatExchange/pull/554/commits/dd4fddd81acb1f8ee34b3f985abbb05652c8cd1c